src/raven.js

```diff
@@ -1320,9 +1323,46 @@ Raven.prototype = {
             exception.value = truncate(exception.value, max);
         }
 
+        var request = data.request;
+        if (request) {
+            if (request.url) {
+                request.url = truncate(request.url, this._globalOptions.maxUrlLength);
+            }
+            if (request.Referer) {
+                request.Referer = truncate(request.Referer, this._globalOptions.maxUrlLength);
+            }
+        }
+
+        if (data.breadcrumbs && data.breadcrumbs.values)
+            this._trimBreadcrumbs(data.breadcrumbs);
+
         return data;
     },
 
+    /**
+     * Truncate breadcrumb values (right now just URLs)
+     */
+    _trimBreadcrumbs: function (breadcrumbs) {
+        // known breadcrumb properties with urls
+        // TODO: also consider arbitrary prop values that start with (https?)?://
+        var urlprops = {to: 1, from: 1, url: 1},
+            crumb,
+            data;
+
+        for (var i = 0; i < breadcrumbs.values.length; i++) {
+            crumb = breadcrumbs.values[i];
+            if (!crumb.hasOwnProperty('data'))
+                continue;
+
+            data = crumb.data;
+            for (var prop in urlprops) {
```

Comment:
@benvinegar late to the party here but this is slightly unsafe - if `Object.prototype` has a property added to it *and* `data` has an own property of that same name, we might truncate something we don't mean to. Simplest fix is to add `&& prop.hasOwnProperty(prop)` but I'm not sure why we don't just make `urlprops` an array and iterate through. Call it one way or the other and I'll make a PR.